components/EpisodeListItem/index.js

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledListItem = styled.li`
+  display: flex;
+  flex-direction: row;
+  gap: 2rem;
+  background-color: whitesmoke;
+  margin: 0.5rem;
+  padding: 0.5rem;
+  border: 1px solid;
```

Comment:
Did you forget to set a border-color or is this intended? 🤔

Reply:
its intended, since I only use the border to make the list items visible (for the moment)